cdap-app-fabric/src/main/java/co/cask/cdap/app/guice/AppFabricServiceRuntimeModule.java

```diff
@@ -318,6 +320,7 @@ protected void configure() {
       bind(ProgramLifecycleService.class).in(Scopes.SINGLETON);
       bind(NamespaceAdmin.class).to(DefaultNamespaceAdmin.class).in(Scopes.SINGLETON);
       bind(NamespaceQueryAdmin.class).to(DefaultNamespaceQueryAdmin.class).in(Scopes.SINGLETON);
+      bind(SecureStoreService.class).to(DefaultSecureStoreService.class);
```

Comment:
That will need for cdap-security to depend on cdap-app-fabric which in turn causes a circular dependency between cdap-watchdog and cdap-notifications.